src/Network/Probe.php

```diff
@@ -143,27 +141,26 @@ private static function ownHost($host)
 	}
 
 	/**
-	 * Probes for webfinger path via "host-meta"
+	 * Probes for webfinger path via 'host-meta'
```

Comment:
So, here's the thing. For strings PHP code accepts single and double quotes (and backticks too) but in regular language, quoted text only accepts double-quotes (or double chevrons in French for example). While replacing double-quotes by single quotes in the PHP code makes sense, it doesn't as much in English comments.
```suggestion
	 * Probes for webfinger path via "host-meta"
```